src/crm/utility/ZohoLoggerInterface.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace zcrmsdk\crm\utility;
+
+interface ZohoLoggerInterface
```

Comment:
That's how it should be done in the first place! Thanks @mikedfunk !

Reply:
@krlv Thanks.

It's still kind of weird to me that there is a logger implementation from within this sdk, but I was trying to fix without refactoring the sdk itself while providing the flexibility to handle logging any way you like, including doing nothing with it.

If I had written this from the beginning I would not have included a logger at all.